packages/framework/resources/views/layouts/docs.blade.php

```diff
@@ -3,8 +3,8 @@
 <head>
     @include('hyde::layouts.head')
 </head>
```

Comment:
@laryhills looks really good, and thanks for adding a screen recording! I appreciate that. One quick thing: Could you explain the reason for changing the classes on the body tag? Do they make an improvement?